src/viam/sdk/common/utils.cpp

```diff
@@ -130,5 +130,18 @@ bool operator==(const response_metadata& lhs, const response_metadata& rhs) {
     return lhs.captured_at == rhs.captured_at;
 }
 
+bool from_dm_from_extra(const AttributeMap& extra) {
+    if (!extra) {
+        return false;
+    }
+    if (auto pos = extra->find("fromDataManagement"); pos != extra->end()) {
```

Comment:
I think this syntax is C++17 specific but the driver targets C++14.

Reply:
thanks for the catch! fixed